src/lasagna/easy.py

```diff
@@ -0,0 +1,162 @@
+from typing import (
+    Callable,
+    List,
+    Type,
+    Any, # FIXME
+)
+import json
+
+from pydantic import BaseModel
+
+from . import (
+    build_simple_agent,
+    Model,
+    EventCallback,
+    EventPayload,
+    AgentRun,
+    ExtractionType,
+    flat_messages,
+    noop_callback,
+    recursive_extract_messages,
+)
+
+
+def simple_text_callback_binder(simple_text_callback: Callable[[str], None]) -> Callable[[EventPayload], None]:
+    async def bound_simple_text_callback(event: EventPayload):
+        if event[0] == 'ai' and event[1] == 'text_event':
+            await simple_text_callback(event[2])
+    return bound_simple_text_callback
+
+
+async def simple_ask(
+    partially_bound_model: Any,
+    system_prompt: str,
+    human_prompt: str,
+    streaming_callback: Callable[[str], None] | None = None,
+    tools: List[Callable] = [],
+    force_tool: bool = False,
+    max_tool_iters: int = 5,
+) -> str:
+    agent = partially_bound_model(
+        build_simple_agent(
+            name = "simple_ask",
+            tools = tools,
+            force_tool = force_tool,
+            max_tool_iters = max_tool_iters,
+        )
+    )
+    runs: list[AgentRun] = [
+        flat_messages(
+            agent_name = "simple_ask",
+            messages = [
+                {
+                    'role': 'system',
+                    'text': system_prompt,
+                },
+                {
+                    'role': 'human',
+                    'text': human_prompt,
+                },
+            ],
+        )
+    ]
+    response: AgentRun = await agent(
+        simple_text_callback_binder(streaming_callback) if streaming_callback else noop_callback,
+        runs,
+    )
+    last_message = response["messages"][-1]
```

Comment:
Above this line you'll have to `assert response['type'] == 'messages'` to make mypy happy. (not tested, just from memory so change as you need)